buildSrc/src/main/java/Config.kt

```diff
@@ -33,7 +33,6 @@ object Config {
 
         val minSdkVersion = 14
         val minSdkVersionOkHttp = 21
-        val minSdkVersionFragment = 21
```

Comment:
@wzieba do you remember why you've set 21 instead of 14?

Reply:
I don't remember and I think it was rather not a deliberative decision - I think I've just copied the value from `minSdkVersionOkHttp`. Sorry!

Reply:
not a problem at all, we're in a beta state so its the right moment to get it fixed :)
thanks again for the contribution @wzieba